pkg/store/bucket.go

```diff
@@ -1472,7 +1483,16 @@ func (r *bucketIndexReader) fetchPostings(groups []*postingGroup) error {
 				r.stats.postingsTouched++
 				r.stats.postingsTouchedSizeSum += len(b)
 
-				_, l, err := r.dec.Postings(b)
+				// Even if this instance is not using compression, there may be compressed
+				// entries in the cache written by other stores.
+				var l index.Postings
```

Comment:
again `var (` available.

Reply:
OK :) Personally I find `var (` ugly

Reply:
Curious why, but yea - not a blocker (:

Reply:
Because of extra indentation. But I've used var block here, as you suggested :)